cis_enforcements/azure/cis_v200/section9/README.md

```diff
@@ -0,0 +1,93 @@
+---
+categories: ["cis"]
+---
+
+# Enforce Azure CIS v2.0.0 - Section 9 - AppService
+
+This section covers security recommendations for Azure AppService.
```

Comment:
```suggestion
# Enforce Azure CIS v2.0.0 - Section 9 - App Service

This section covers security recommendations for Azure App Service.
```

I can see the CIS doc mentions `AppService`. But let's follow the actual Azure docs and use `App Service`.